docs/run-platform-server/configuring.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: Configuring
+sidebar_position: 30
+---
+
+import { Alert } from "@site/src/components/Alert";
+
+Pre-requisites:
+
+- You have identified the [Installation](./installing.mdx) method for host system:
+
+  For bare metal, you have two executables: `stellar-horizon` and `stellar-core` both on host OS' `$PATH`.
+
+  For running horizon image with docker daemon, you will use the [stellar/stellar-horizon](https://hub.docker.com/r/stellar/stellar-horizon) hosted on dockerhub. You have already pulled the stellar/stellar-horizon image via `docker pull stellar/stellar-horizon:<tag_version>` onto host. This image contains the `stellar-horizon` and `stellar-core` within.
+
+  For Kubernetes with [Horizon Helm Chart](https://github.com/stellar/helm-charts/tree/main/charts/horizon), you have followed the [Install Horizon with Helm Chart](./installing.mdx#helm-chart-installation).
+
+- [Initialize Database](#initialize-horizon-database)
+
+You are now ready to identify the configuration parameters needed to perform three important, distinct roles:
+
+- **serving requests** via a regular web-based HTTP API, and
+- **ingesting ledgers** from core nodes of a Stellar network to keep its world-view up to date, and
+- **transaction submission** for sending transactions to the Stellar network.
```

Comment:
if we are encouraging tx-sub via captive core then the tx-sub role is no longer distinct from the ingestion role

Reply:
"You are now ready to identify the configuration parameters needed to perform three important, distinct roles:"

Reply:
got it, thanks - https://github.com/stellar/stellar-docs/pull/249/commits/dadd4f9ee2401d97279abc7c98c723c306118b4e